bokeh/models/tools.py

```diff
@@ -174,6 +174,23 @@ class PanTool(Drag):
     """)
 
 
+class WheelPanTool(Scroll):
+    """ *toolbar icon*: |wheel_pan_icon|
+
+    The wheel pan tool allows the user to pan the plot along the configured
+    dimension using the scroll wheel.
+
+    .. |wheel_pan_icon| image:: /_images/icons/WheelPan.png
+        :height: 18pt
+
+    """
+
+    dimension = Enum(Dimension, default="width", help="""
+    Which dimension the wheel pan tool is constrained to act in. By
+    default the wheel pan tool will pan the plot along the x-axis.
+    """)
```

Comment:
Maybe ought to explicitly state that the y-axis can be selected using "height"?  It's obvious, but still, it's an Enum, and it would be nice to be clear what the enumerated options are...